Moab.Models/Helpers/ExerciseImportHelper.cs

```diff
@@ -20,17 +22,159 @@ public class ExerciseImportHelper
+            List<string[]> ImportList = SplitCSVInput(importCSV);
+
+
             return false;
         }
 
         #endregion
 
+        #region test code for private methods  **REMOVE FROM PRODUCTION CODE**
+
+        public string[] TestLineSplit(string Line)
+        {
+            return SplitCSVLine(Line);
+        }
+
+        public List<string[]> TestInputProcessing(string input)
+        {
+            return SplitCSVInput(input);
+        }
+
+        public bool TestIsHeaderValid(string header)
+        {
+            return IsHeaderValid(header);
+        }
+
+        #endregion
+
         #region Private / Protected Methods
         protected bool IsHeaderValid(string header)
         {
+            string checkHeader = "ExerciseCode,Name,CDT_Class,CDT_AtHome,IsMovementDataCollected,UnitTarget,HintEasier,HintHarder,Hint1,Hint2,MDT_Class,MDT_AtHome,OldCode,Name_animationFile,Old_Name_animationFile";
+            if (header == checkHeader)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        protected Exercise FindExtantExerciseInCollection(ICollection<Exercise> exercises, string exerciseCode)
+        {
+           foreach (Exercise exercise in exercises)
+            {
+                if (exercise.ExerciseCode == exerciseCode)
+                {
+                    return exercise;
+                }
+            }
+            return null;
+        }
+
+        protected void UpdateExercise(Exercise exercise, string[] updateCSV)
+        {
+            //TODO: add support for  generic hint collection
+            exercise.ExerciseCode = updateCSV[(int)ExerciseCSVColumns.ExerciseCode];
+            exercise.Name = updateCSV[(int)ExerciseCSVColumns.Name];
+            exercise.EasierHint = updateCSV[(int)ExerciseCSVColumns.HintEasier];
+            exercise.HarderHint = updateCSV[(int)ExerciseCSVColumns.HintHarder];
+            exercise.HasRepetitionTarget = ConvertYNtoBool(updateCSV[(int)ExerciseCSVColumns.UnitTarget]);
+            exercise.DateLastUpdated = DateTime.Now;
+        }
+
+        // Converts Y into true and N into false
+        protected bool ConvertYNtoBool(string input)
+        {
+            if (input.ToUpper() == "Y")
+            {
+                return true;
+            }
+            else if (input.ToUpper() == "N")
+            {
+                return false;
+            }
             return false;
         }
 
+        // Converts the input string into a
+        protected List<string[]> SplitCSVInput(string CSVInput)
+        {
+            // Create LineList
+            List<string[]> LineList = new List<string[]>();
+            int iterator = 0;
+            do
+            {
+                string[] line = new string[14];
+                int iteratorNext = CSVInput.IndexOf('\n', iterator);
```

Comment:
Use Environment.Newline instead of '\n'.  This will help your code run correctly regardless of computer environment it is in.
Using string.Split(Environment.Newline) you can replace lines 115 - 125 in one line of code.